controllers/collection_exercise_controller.py

```diff
@@ -165,7 +166,9 @@ def create_and_execute_collection_exercise(survey_id, period, user_description,
                                                     'resources/collection_instrument_files/064_201803_0001.xlsx')
     time.sleep(5)
     execute_collection_exercise(survey_id, period)
-    poll_database_for_iac(survey_id, period)
+    iac = poll_database_for_iac(survey_id, period)
```

Comment:
Looks like some changes are required to this poll function so that it returns an iac. Currently it just stops looping once it's found one.

Reply:
Something like:

```
    while True:
        iac = database_controller.get_iac_for_collection_exercise(collection_exercise_id)
        if iac:
            logger.info('Collection exercise finished executing', survey_id=survey_id, period=period)
            break
        time.sleep(5)
    return iac
```

Reply:
fixed, sorry code was there but had some debug code in that file and in a different changeset that I didn't commit, should be good now though

Reply:
I'll give it another whirl :)